src/app/organizations/settings/account.component.ts

```diff
@@ -101,6 +105,9 @@ export class AccountComponent {
     async deleteOrganization() {
         await this.modalService.openViewRef(DeleteOrganizationComponent, this.deleteModalRef, comp => {
             comp.organizationId = this.organizationId;
+            comp.onSuccess.subscribe(() => {
+                this.router.navigate(['/']);
+            });
```

Comment:
Was this changed to fix something?

Reply:
Not fix but to abstract the `DeleteOrganizationComponent` a little more. We also needed to use it on the `FamiliesForEnterpriseSetupComponent` but it was hardcoded to navigating to the vault afterwards. Adding in a event that can be subscribed to depending on the consumer allowed us to reuse the component with minimal changes.